.github/workflows/client-test-en.yaml

```diff
@@ -0,0 +1,56 @@
+name: Client Tests with EN
+on:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - 'api/**'
+      - 'client/exposure_notifications/**'
+      - 'client/app/**'
+      - 'client/app_en/**'
+      - 'client/flutter/**'
```

Comment:
Same comment on tests... these are faster at ~2 min/run, but every min counts in our CI...

Reply:
Because the tests run in parallel with the other tests , and because tests take strictly less time than the build it appears to cause no added wait time.  See:
<img width="343" alt="Screen Shot 2020-05-04 at 9 29 48 AM" src="https://user-images.githubusercontent.com/11729521/80989756-3b997d80-8dea-11ea-8834-2cdcd966a5be.png">